continuous_integration/setup_conda_environment.cmd

```diff
@@ -30,7 +30,7 @@ call deactivate
     joblib ^
     jupyter_client ^
     mock ^
-    msgpack-python ^
+    msgpack ^
```

Comment:
So this part is using `conda` where it is still named `msgpack-python`. Seems to be causing the CI failure. Reverting this part should fix that issue.

Reply:
I would have guessed this is only used in Windows though, so that does not really explain the Travis failure. I pushed the change anyway.